FEEDBACK: Bottom Navigation Bar background color should be white in light mode

basitali1509 opened the pull request:
### **What does this PR do?**

This PR fixes the background color of the bottom navigation bar from black to white in the light mode for a better UI.

fixes #25 

### **Type of change**

- Simple Media Query logic added to check if dark mode is active.

### **Updated Output** 

<img width="248" alt="image" src="https://github.com/dstark5/Openlib/assets/84122396/293bdb37-9312-4823-9f33-79e98fde0b85">                     <img width="254" alt="image" src="https://github.com/dstark5/Openlib/assets/84122396/86d4ca84-7df8-4e57-81d4-03584cb735e0">

dstark5 commented:
Hello,
Thanks for your effort
Kindly choose color to differentiate the bottom nav bar in light mode,So that I can merge the PR

basitali1509 commented:
> Hello, Thanks for your effort Kindly choose color to differentiate the bottom nav bar in light mode,So that I can merge the PR

I have updated the code and updated the output screen shot kindly check. As you said earlier to try grey color so I used grey with shade 300 to make the bottom navbar prominent.

dstark5 commented:
PR Merged
